integration-tests/mongodb-panache/src/main/java/io/quarkus/it/mongodb/panache/test/TestImperativeEntity.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.it.mongodb.panache.test;
+
+import io.quarkus.mongodb.panache.PanacheMongoEntity;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
```

Comment:
Isn't that automatic? Shouldn't it be?

Reply:
This should be fixed by #8211 I'll test to remove it as the PR is fixed and send another PR

Reply:
Right, great.